### PR TITLE
(Binary Serialization Part 0) Updated to latest version of FlatBuffers.

### PR DESCRIPTION
**Pull request by dgant**
https://github.com/TorchCraft/TorchCraft/issues/213

I'll be issuing a series of pull requests which cover the full transition to binary serialization.

---

Updating to FlatBuffers 1.7.1.
* Updated relevant FlatBuffers headers (it requires two more)
* Updated flatc invocation script to verify that we're using the correct version of FlatBuffers

@jgehring @ebetica @syhw

### CODE REVIEW

File: BWEnv/fbs/messages_generated.h [hunk 1]
```diff
@@ -57,6 +57,20 @@ enum class Any : uint8_t {
   MAX = Error
```
Comment (dgant) [inline BWEnv/fbs/messages_generated.h]: This is the autogenerated code produced by flatc.

File: BWEnv/fbs/base.h [hunk 2]
```diff
@@ -0,0 +1,227 @@
+#ifndef FLATBUFFERS_BASE_H_
```
Comment (dgant) [inline BWEnv/fbs/base.h]: One of the two new flatbuffers headers.

File: BWEnv/fbs/stl_emulation.h [hunk 3]
```diff
@@ -0,0 +1,220 @@
+/*
```
Comment (dgant) [inline BWEnv/fbs/stl_emulation.h]: This is the other new flatbuffers header.

**Comment by jgehring** [comment]
Great, thanks for splitting up the PR!

**Pull request merged**